docs/index.html

```diff
@@ -1,64 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="UTF-8" />
-		<title>
-			zwave-js - Z-Wave driver written entirely in JavaScript/TypeScript
-		</title>
-		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-		<meta
-			name="description"
-			content="Z-Wave driver written entirely in JavaScript/TypeScript"
-		/>
-		<meta
-			name="viewport"
-			content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-		/>
-		<link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/vue.css" />
-		<style>
-			.markdown-section h1 code,
-			.markdown-section h2 code,
-			.markdown-section h3 code,
-			.markdown-section h4 code,
-			.markdown-section h5 code,
-			.markdown-section h6 code {
-				/* Reset the font size to the headline size */
-				font-size: inherit;
-			}
-		</style>
-	</head>
-	<body>
-		<div id="app"></div>
-		<script src="//unpkg.com/docsify/lib/docsify.min.js"></script>
-		<script src="//unpkg.com/docsify-edit-on-github/index.js"></script>
-		<!-- Load syntax highlighters -->
-		<script src="//unpkg.com/prismjs/components/prism-bash.min.js"></script>
-		<script src="//unpkg.com/prismjs/components/prism-typescript.min.js"></script>
-		<script src="//unpkg.com/prismjs/components/prism-json.min.js"></script>
 
```

Comment:
doesn't seem like anything changed here - can you reset to the commited version?

Reply:
Done